packages/schema/README.md

```diff
@@ -1130,6 +1130,14 @@ S.literal(2n); // bigint literal
 S.literal(true);
 ```
 
+We can also `pickLiteral` from a literal schema.
```

Comment:
I would add an explanation of the use case, that is, the motivation for why a user should use `S.literal("a", "b", "c").pipe(S.pickLiteral("a", "b"))` instead of simply using `S.literal("a", "b")`, it's not at all obvious at first glance.

Reply:
Proposal: (I would add the following bloc under the current version)
Sometimes we need to reuse schemas in other parts of the code. Let's see an example:
```ts
const FruitId = Schema.number;
// the source of truth regarding the Fruit category
const FruitCategory = Schema.literal("sweet", "citrus", "tropical"); 

class Fruit extends Schema.Class<Fruit>()({
  id: FruitId,
  category: FruitCategory,
}) {}

//Here we want to reuse are FruitCategory definition to create a sub type of the Fruit.
class SweetAndCitrusFruit extends Schema.Class<SweetAndCitrusFruit>()(
  {
    fruitId: FruitId,
    category: FruitCategory.pipe(Schema.pickLiteral("sweet", "citrus")), 
    /* 
    By using pickLiteral from the FruitCategory, we ensure that the values selected 
    are those defined in the category definition above. 
    If we remove "sweet" from the FruitCategory definition, TypeScript will notify us.
    */
  }
) {}
```
ps. the **`Literals`** section is located at the beginning of the `README.md`, should we consider using `Schema.Class`, `Schema.struct` or both in the proposal above?

Reply:
LGTM 

(only a typo in "//Here we want to reuse **are** FruitCategory definition to create a sub type of the Fruit", I'd propose "Here, we want to reuse our FruitCategory definition to create a subtype of Fruit")

p.s.
let's use `Schema.struct`